services/workflows-service/scripts/alerts/generate-transactions.ts

```diff
@@ -80,7 +80,7 @@ export const generateTransactions = async (
   }> = [];
 
   // Create transactions with a random counterparty ID for each
-  for (let i = 0; i < 1000; i++) {
```

Comment:
Locally I consistently get a transaction timeout. Will revert once everything works in dev.